test/e2e/feature/feature.go

```diff
@@ -357,6 +357,9 @@ var (
 	// (used for testing fine-grained SupplementalGroups control <https://kep.k8s.io/3619>)
 	SupplementalGroupsPolicy = framework.WithFeature(framework.ValidFeatures.Add("SupplementalGroupsPolicy"))
 
+	// TODO: document the feature (owning SIG, when to use this feature for a test)
```

Comment:
You can now add an comment to it

Reply:
This is the last comment to address in this PR. Please write something like: 

> "tests marked with this feature MUST run with the CRI Proxy configured so errors can be injected into the kubelet's CRI calls."

Reply:
Just find this needs an update.
/lgtm cancel